### PR TITLE
feat: Add team page

### PR DESCRIPTION
**Pull request by Haider8**
Details of all the members are fetched from the database. Following information will be shown on the 'team' page:
- Avatar

- Name

- Position

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Amazing work @Haider8. Let's polish the implementation and merge it. 

First, it needs to be sorted according to the rank and contact information needs to be shown.

Really good work

**Comment by iamareebjamal** [comment]
Even though you have correctly added a getter with correct method, it is not sorted according to rank. I will check why is this happening

**Comment by Haider8** [comment]
thank you @iamareebjamal :)

**Comment by Haider8** [comment]
On the localhost it is well sorted, but something isn't okay fine when it is hosted. @iamareebjamal

**Comment by iamareebjamal** [comment]
It's not working for me locally as well

**Comment by iamareebjamal** [comment]
For me, it is still unsorted

**Comment by iamareebjamal** [comment]
Just change the `>` sign in sort method to `-`

**Comment by Haider8** [comment]
okay :+1: @iamareebjamal

**Comment by iamareebjamal** [comment]
@Haider8 Instead of binding team/current/members. Bind team/current and show the contact links

File: src/components/Other/Team.vue [hunk 1]
```diff
@@ -67,12 +82,48 @@ export default {
...
   computed: {
-    ...mapGetters('team', ['team'])
+    ...mapGetters('team', ['members'])
+  },
+  methods: {
+    iconUrl: function(type) {
+      switch (type) {
+        case 'facebook':
+          return 'https://cdn3.iconfinder.com/data/icons/free-social-icons/67/facebook_circle_color-512.png'
+          break
```
Comment (iamareebjamal) [inline src/components/Other/Team.vue]: No need of break statements when returning

File: src/components/Other/Team.vue [hunk 2]
```diff
@@ -67,12 +82,48 @@ export default {
...
+      if (
+        type == 'facebook' ||
+        type == 'linkedin' ||
+        type == 'twitter' ||
+        type == 'g-plus'
+      ) {
+        window.location.href = link
+      } else {
+        window.alert(link)
+      }
```
Comment (iamareebjamal) [inline src/components/Other/Team.vue]: Instead of this method, use anchor tag. `<a href="url"></a>`

File: package.json [hunk 3]
```diff
@@ -8,6 +8,9 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.12",
+    "@fortawesome/free-solid-svg-icons": "^5.6.3",
+    "@fortawesome/vue-fontawesome": "^0.1.4",
```
Comment (iamareebjamal) [inline package.json]: These shouldn't be needed

**Comment by iamareebjamal** [comment]
Also, in case of no contact, there is double shadow

File: src/components/Other/Team.vue (hunk 2 above)
Comment (Haider8) [reply src/components/Other/Team.vue]: Anchor tag isn't working with <v-img>

Comment (iamareebjamal) [reply src/components/Other/Team.vue]: Wrap the v-btn in anchor

**Pull request merged**